platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/widgets/MyLocationView.java

```diff
@@ -702,7 +702,7 @@ void updateLatLng(@NonNull Location location) {
                     builder.bearing(location.getBearing());
                 }
                 gpsDirection = location.getBearing();
```

Comment:
I followed up this request with some other cleanup and corrections, want to review again?